### PR TITLE
Serial indicator blinks when receiving data

### PR DESCRIPTION
**Pull request by cait1inhennessy**

### CODE REVIEW

File: webapp/src/serialindicator.tsx [hunk 1]
```diff
@@ -12,36 +12,43 @@ export interface SerialIndicatorProps {
...
+        this.setState({ active: false, receivingData: false})
     }
 
     render() {
         if (!this.state.active) return <div />;
         return <div className="ui segment inverted serialindicator" tabIndex={0} onClick={this.props.onClick} onKeyDown={sui.fireClickOnEnter}>
             <div className="ui label circular">
                 <div className="detail indicator">
-                    <span className="ui green empty circular label" />
+                    <span className={` ${this.state.receivingData ? "blink" : ""} ui green empty circular label`} />
```
Comment (microsoftsam) [inline webapp/src/serialindicator.tsx]: nit: extra space before the ${this.state}

File: webapp/src/serialindicator.tsx [hunk 2]
```diff
@@ -12,36 +12,43 @@ export interface SerialIndicatorProps {
...
+        if (msg.type === "serial") {
             const sim = !!msg.sim
             if (sim === this.props.isSim) {
-                this.setState({ active: true })
+                if (!this.state.active) {
+                    this.setState({ active: true, receivingData: true })
+                } else {
+                    clearTimeout(this.animationStopper)
+                }
+                this.animationStopper = setTimeout(() => {this.setState({receivingData: false})}, 1000)
```
Comment (microsoftsam) [inline webapp/src/serialindicator.tsx]: Clear the animation onComponentUnmount?

Comment (pelikhan) [reply webapp/src/serialindicator.tsx]: Use ``Util.debounce`` to debounce ``receivingData: false``

File: webapp/src/serialindicator.tsx (hunk 1 above)
Comment (caitlin-hennessy) [reply webapp/src/serialindicator.tsx]: Fixed :)

**Comment by pelikhan** [comment]
I think we can close this.

**Comment by cait1inhennessy** [comment]
Closing due to decision to use bar chart gif instead of blinking indicator.

**Pull request closed without merging**